app/helpers/govuk_list_helper.rb

```diff
@@ -0,0 +1,36 @@
+module GovukListHelper
+  def govuk_list(array = nil, type: nil, spaced: nil, classes: nil, html_attributes: {}, &block)
+    html_classes = ["#{brand}-list"]
+
+    case type.to_s.to_sym
+    when :bullet
+      html_classes << ["#{brand}-list--bullet"]
+      tag_type = "ul"
+    when :number
+      html_classes << ["#{brand}-list--number"]
+      tag_type = "ol"
```

Comment:
Technically you _could_ have numbers for an ordered list, or bullets for an ordered list (the Design System doesn’t have a Nunjucks macro for this, but the classes would work) - however I don’t think there’s any real use case where this would make sense?

Reply:
Yeah, I'd agree. Would having these as separate named args `bullets: true` and `numbers: true` make it clearer? It could `fail` when both are true and gets rid of the need to do the symbol/string conversions.

Reply:
Hmm, interesting. I think that makes it less clear, at least to me? Having a single `type` argument more closely implies that it can only be bullets OR numbers?